Misc/Helpers.cs

```diff
@@ -43,7 +46,12 @@ public static IWebDriver InitDriver()
             service.SuppressInitialDiagnosticInformation = true;
             ChromeOptions options = new ChromeOptions();
 
-            options.AddArgument(@"user-data-dir=C:\Users\" + Buffer.ToString() + @"\AppData\Local\Google\Chrome\User Data");
+            string user_data_dir = String.Format(@"C:\Users\{0}\AppData\Local\{1}\User Data", Buffer.ToString(), BrowserRouting);
```

Comment:
NTH: It's usually better to use the environment variable `%LOCALAPPDATA%` but since it was already hard-coded `C:\Users` then I guess it's fine